app/controllers/admin/users_controller.rb

```diff
@@ -37,9 +37,11 @@ def show
       @user = User.find(params[:id])
 
       if FeatureFlag.enabled?(:admin_member_view)
+        set_current_tab(params[:tab])
         set_feedback_messages
         set_related_reactions
       end
+      # return set_user_details unless FeatureFlag.enabled?(:admin_member_view)
```

Comment:
@juliannatetreault is there any action item for this comment?

Reply:
Nope! Removed; great catch. 😊